docs/content/users/usage/cli.md

```diff
@@ -56,7 +56,7 @@ ddev export-db >/tmp/db.sql.gz
 
 ## `ddev import-files`
 
-To import static file assets for a project, such as uploaded images and documents, use the command [`ddev import-files`](../usage/commands.md#import-files). This command will prompt you to specify the location of your import asset, then import the assets into the project’s upload directory. To define a custom upload directory, set the [`upload_dir`](../configuration/config.md#upload_dir) config option. If no custom upload directory is defined, the default will be used:
+To import static file assets for a project, such as uploaded images and documents, use the command [`ddev import-files`](../usage/commands.md#import-files). This command will prompt you to specify the location of your import asset, then import the assets into the project’s upload directory. To define a custom upload directory, set the [`upload_dirs`](../configuration/config.md#upload_dirs) config option. If no custom upload directory is defined, the default will be used:
 
 * For Drupal projects, this is the `sites/default/files` directory.
```

Comment:
Update to show this -> these.

Also, with Drupal we might want to make upload_dirs default with private files. We could have it default to have both sites/default/files and ../private , and add the settings to settings.ddev.php

Reply:
Add Craft CMS - where these are configured independently in the various projects. So we need to say "You have to config yourself" 

Not sure about Laravel... is there something we should say here about Laravel @tyler36 ?

Reply:
Done